.eslintrc.js

```diff
@@ -58,5 +61,12 @@ module.exports = {
         'node/no-missing-require': 0,
       },
     },
+    {
+      files: ['**/*.md/*.js'],
+      rules: {
+        'unicorn/filename-case': 0,
+        'node/no-unpublished-require': 0,
```

Comment:
Should we add those rules to `@netlify/eslint-config-node`?

Reply:
Done in https://github.com/netlify/eslint-config-node/pull/112